cms/templates/cms/program_page.html

```diff
@@ -104,11 +104,37 @@
     </div>
   </div>
 
+  {# if page.faculty_members.count > 0 #}
```

Comment:
`{#` is a comment in Django templates -- this logic was commented-out for testing. Now that the PR is ready for review, it is no longer commented out.